inst/qml/ClassicProcess.qml

```diff
@@ -327,6 +327,13 @@ Form
 								name: "pathCoefficients"
 								label: qsTr("Paths")
 								checked: pathCoefficientsForAllModels.checked
+								
+								CheckBox
+  							{
+  								name: "intercepts"
+  								label: qsTr("Intercepts")
+  								checked: interceptsForAllModels.checked
+  							}
```

Comment:
```suggestion
  							    {
  								    name: "intercepts"
  								    label: qsTr("Intercepts")
  								    checked: interceptsForAllModels.checked
  							    }
```

Just a QML indentation fix.

Reply:
I don't understand why but in RStudio the indentation shows up fine, but in Notepad++ I see the problem you're seeing.